samples/SkijaInjectSample/src/main/kotlin/SkijaInjectSample/App.kt

```diff
@@ -15,6 +15,7 @@ import kotlin.math.cos
 import kotlin.math.sin
 
 fun main(args: Array<String>) {
+    // System.setProperty("skiko.renderApi", "RASTER")
```

Comment:
Please remove.

Reply:
Or maybe move to Gradle script in commented out form.

Reply:
I think it is better to remove this because this is only for software rendering testing.